yt/frontends/swift/data_structures.py

```diff
@@ -0,0 +1,209 @@
+"""
+Data structures for Swift frontend
+
+
+
+
+"""
+from __future__ import print_function
+
+#-----------------------------------------------------------------------------
+# Copyright (c) 2018, yt Development Team.
+#
+# Distributed under the terms of the Modified BSD License.
+#
+# The full license is in the file COPYING.txt, distributed with this software.
+#-----------------------------------------------------------------------------
+
+import numpy as np
+
+from yt.utilities.on_demand_imports import _h5py as h5py
+
+from uuid import uuid4
+
+from yt.utilities.logger import ytLogger as mylog
+
+from yt.frontends.sph.data_structures import \
+    SPHDataset, \
+    SPHParticleIndex
+
+from .fields import SwiftFieldInfo
+
+from yt.data_objects.static_output import \
+    ParticleFile
+
+from yt.funcs import only_on_root
```

Comment:
super minor nit: can you group the imports together in logical sections with no line breaks? First standard library and external library imports, then internal yt imports. Like this:

```
import numpy as np
from yt.utilities.on_demand_imports import _h5py as h5py
from uuid import uuid4

from yt.utilities.logger import ytLogger as mylog
from yt.frontends.sph.data_structures import \
    SPHDataset, \
    SPHParticleIndex
from .fields import SwiftFieldInfo
from yt.data_objects.static_output import \
    ParticleFile
from yt.funcs import only_on_root
```

Reply:
ultimately I want to start using [black](https://github.com/ambv/black) to automate discussions like this but until then...

Reply:
yeah! When do you propose to start using black? Josh always talks about this - it looks really cool